web-registry/src/components/organisms/BasketOverview.tsx

```diff
@@ -0,0 +1,268 @@
+                    // TODO: harcoded url for curator
+                    link={'https://www.regen.network/'}
+                  />
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                  <Item
+                    label="allowed credit classes"
+                    data={allowedCreditClasses}
+                    // TODO: url for credit class
+                    link={'/credit-classes/'}
+                  />
+                </Grid>
+                {minStartDate && (
+                  <Grid item xs={12} sm={6}>
+                    <Item
+                      label="min start date"
+                      data={formatDate(minStartDate)}
+                    />
+                  </Grid>
+                )}
+                {startDateWindow && (
+                  <Grid item xs={12} sm={6}>
+                    <Item
+                      label="start date window"
+                      data={formatDate(startDateWindow)}
+                    />
+                  </Grid>
+                )}
+              </Grid>
+            </OnBoardingCard>
+          </Grid>
+        </Grid>
+      </Section>
+    </Box>
+  );
+};
+
+export { BasketOverview };
+
+/**
+ * Basket summary item
+ */
+
+const useStylesItem = makeStyles(theme => ({
+  label: {
+    fontSize: theme.typography.pxToRem(12),
+    color: theme.palette.primary.contrastText,
+    letterSpacing: '1px',
+    lineHeight: '15px',
+    marginBottom: theme.spacing(2),
+  },
+  data: {
+    fontSize: theme.typography.pxToRem(16),
+    fontWeight: 400,
+    '& a': {
+      color: theme.palette.text.secondary,
+      fontSize: theme.typography.pxToRem(16),
+      fontWeight: 400,
+    },
+  },
+  arrowIcon: {
+    marginLeft: theme.spacing(1),
+    marginBottom: theme.spacing(0.3),
+    height: 9,
+    width: 13,
+  },
+}));
+
+interface ItemProps {
+  label: string;
+  data: string | string[];
+  link?: string;
+}
+
+const Item = ({ label, data, link }: ItemProps): JSX.Element => {
+  const styles = useStylesItem();
+
+  return (
+    <Box sx={{ mt: 4 }}>
+      <Label className={styles.label}>{label}</Label>
+      {Array.isArray(data) ? (
+        data.map(item => (
+          <Description key={`basket-${item}`} className={styles.data}>
+            {link ? (
+              <Link href={link + item} target="_blank" rel="noreferrer">
+                {parseText(item)}
+                <SmallArrowIcon className={styles.arrowIcon} />
+              </Link>
+            ) : (
+              parseText(item)
+            )}
+          </Description>
+        ))
+      ) : (
+        <Description className={styles.data}>
+          {link ? (
+            <Link href={link} target="_blank" rel="noreferrer">
+              {parseText(data)}
+              <SmallArrowIcon className={styles.arrowIcon} />
+            </Link>
+          ) : (
+            parseText(data)
+          )}
+        </Description>
+      )}
+    </Box>
+  );
```

Comment:
if you force `data` into an array, you can DRY up the JSX:
```tsx
const itemData = Array.isArray(data) ? data : [data];
```
then you could 
```suggestion
      {itemData.map((item, i) => (
        <Description key={`basket-${item}`} className={styles.data}>
          {link ? (
            <Link href={link + item} target="_blank" rel="noreferrer">
              {parseText(item)}
              <SmallArrowIcon className={styles.arrowIcon} />
            </Link>
          ) : (
            parseText(item)
          )}
        </Description>
      ))}
```

Reply:
super nice! thanks David!